src/main.rs

```diff
@@ -36,12 +36,18 @@ fn main_impl() -> anyhow::Result<()> {
             let root = package.manifest_path.parent().unwrap();
 
             // Process each file in the package
-            // Only look in src so that the root crate does not pick workspace member sources.
```

Comment:
Reading this comment again, I think a more robust solution would be to filter out the workspace members (at `{root}/{member}`), rather than hardcoding the paths where we expect to find the source code.

Reply:
I have implemented this in faa6ac8262216bb1dcc2ff955bc6651e7b1739be